index.html

```diff
@@ -363,7 +365,7 @@ <h2>Textbook</h2>
 <h3><i>Alternate textbooks</i></h3>
 <p>
 
-<a href="http://www.springer.com/us/book/9783642201431"><b>Robotics, Vision and Control: Fundamental Algorithms in MATLAB</b></a>
+<a href="http://www.springer.com/gp/book/9783319544120"><b>Robotics, Vision and Control: Fundamental Algorithms in MATLAB</b></a>
```

Comment:
This link was broken, but the new link seems to go to a new edition of the textbook. Is that okay? Should I find a link to the 2011 edition specifically? Or should we update the publication date here on the website to 2017?